crates/encoding/src/estimate.rs

```diff
@@ -0,0 +1,288 @@
+// Copyright 2024 The Vello authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
Since #456 we have standard copyright headers. Please update this file to have consistent capitalization with the standard.

```suggestion
// Copyright 2024 the Vello Authors
// SPDX-License-Identifier: Apache-2.0 OR MIT
```